tests/unit/iontests.js

```diff
@@ -142,88 +82,282 @@ define(['intern', 'intern!object', 'intern/dojo/node!fs', 'intern/dojo/node!path
+    'ion-tests/iontestdata/bad/annotationLengthTooLongContainer.10n',
+    'ion-tests/iontestdata/bad/annotationLengthTooLongScalar.10n',
+    'ion-tests/iontestdata/bad/annotationLengthTooShortContainer.10n',
+    'ion-tests/iontestdata/bad/annotationLengthTooShortScalar.10n',
+    'ion-tests/iontestdata/bad/annotationNested.10n',
+    'ion-tests/iontestdata/bad/annotationWithNoValue.10n',
+    'ion-tests/iontestdata/bad/blobLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/clobLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/decimalLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/emptyAnnotatedInt.10n',
+    'ion-tests/iontestdata/bad/floatLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/localSymbolTableImportNullMaxId.ion',
+    'ion-tests/iontestdata/bad/minLongWithLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/negativeIntZero.10n',
+    'ion-tests/iontestdata/bad/negativeIntZeroLn.10n',
+    'ion-tests/iontestdata/bad/nopPadWithAnnotations.10n',
+    'ion-tests/iontestdata/bad/stringLenTooLarge.10n',
+    'ion-tests/iontestdata/bad/structOrderedEmpty.10n',
+    'ion-tests/iontestdata/bad/timestamp/timestampHourWithoutMinute.10n',
+]);
+
+let eventSkipList = toSkipList([
+    'ion-tests/iontestdata/good/allNulls.ion',
+    'ion-tests/iontestdata/good/annotationQuotedFalse.ion',
+    'ion-tests/iontestdata/good/annotationQuotedNan.ion',
+    'ion-tests/iontestdata/good/annotationQuotedNegInf.ion',
+    'ion-tests/iontestdata/good/annotationQuotedNull.ion',
+    'ion-tests/iontestdata/good/annotationQuotedNullInt.ion',
+    'ion-tests/iontestdata/good/annotationQuotedOperator.ion',
+    'ion-tests/iontestdata/good/annotationQuotedPosInf.ion',
+    'ion-tests/iontestdata/good/annotationQuotedTrue.ion',
+    'ion-tests/iontestdata/good/clobWithNonAsciiCharacter.10n',
+    'ion-tests/iontestdata/good/clobs.ion',
+    'ion-tests/iontestdata/good/commentMultiLineThenEof.ion',
+    'ion-tests/iontestdata/good/commentSingleLineThenEof.ion',
+    'ion-tests/iontestdata/good/decimal64BitBoundary.ion',
+    'ion-tests/iontestdata/good/decimal_e_values.ion',
+    'ion-tests/iontestdata/good/decimal_values.ion',
+    'ion-tests/iontestdata/good/decimalsWithUnderscores.ion',
+    'ion-tests/iontestdata/good/floatSpecials.ion',
+    'ion-tests/iontestdata/good/floatsWithUnderscores.ion',
+    'ion-tests/iontestdata/good/innerVersionIdentifiers.ion',
+    'ion-tests/iontestdata/good/intBigSize1201.10n',
+    'ion-tests/iontestdata/good/intBigSize13.10n',
+    'ion-tests/iontestdata/good/intBigSize14.10n',
+    'ion-tests/iontestdata/good/intBigSize16.10n',
+    'ion-tests/iontestdata/good/intBigSize256.10n',
+    'ion-tests/iontestdata/good/intBigSize256.ion',
+    'ion-tests/iontestdata/good/intBigSize512.ion',
+    'ion-tests/iontestdata/good/intBinary.ion',
+    'ion-tests/iontestdata/good/intLongMaxValuePlusOne.10n',
+    'ion-tests/iontestdata/good/intLongMinValue.10n',
+    'ion-tests/iontestdata/good/integer_values.ion',
```

Comment:
comments from the previous skiplist should be moved over instead of left as a copy paste below

Reply:
i don't have an easy way to do this.  current plan is to regenerate the skiplists with a perl script on a weekly basis